src/dao/AbstractContractDAO.js

```diff
@@ -21,6 +21,8 @@ const timestampStart = Date.now()
 let events = []
 
 class AbstractContractDAO {
+  converter = converter
```

Comment:
isEmptyAddress removed from converter, but var name stayed as 'converter' for code readability. Uglify will do it work for build.